test/unit/view/drag-handle.spec.js

```diff
@@ -2522,72 +2522,406 @@ describe('drag handle', () => {
             ...Object.keys(map).map((tagName: string) => tagName.toUpperCase()),
           ];
 
-          it('should not start a drag if the target is an interactive element', () => {
-            mixedCase(interactiveTagNames).forEach((tagName: string) => {
-              const element: HTMLElement = document.createElement(tagName);
-              const options = {
-                target: element,
-              };
+          describe('interactive elements', () => {
+            it('should not start a drag if the target is an interactive element', () => {
+              mixedCase(interactiveTagNames).forEach((tagName: string) => {
+                const element: HTMLElement = document.createElement(tagName);
+                const options = {
+                  target: element,
+                };
 
-              control.preLift(wrapper, options);
-              control.lift(wrapper, options);
+                control.preLift(wrapper, options);
+                control.lift(wrapper, options);
 
-              expect(callbacksCalled(callbacks)({
-                onLift: 0,
-              })).toBe(true);
+                expect(callbacksCalled(callbacks)({
+                  onLift: 0,
+                })).toBe(true);
+              });
             });
-          });
 
-          it('should start a drag on an interactive element if asked to by user', () => {
-            // allowing dragging from interactive elements
-            wrapper.setProps({ canDragInteractiveElements: true });
+            it('should start a drag on an interactive element if asked to by user', () => {
+              // allowing dragging from interactive elements
+              wrapper.setProps({ canDragInteractiveElements: true });
 
-            mixedCase(interactiveTagNames).forEach((tagName: string, index: number) => {
-              const element: HTMLElement = document.createElement(tagName);
-              const options = {
-                target: element,
-              };
+              mixedCase(interactiveTagNames).forEach((tagName: string, index: number) => {
+                const element: HTMLElement = document.createElement(tagName);
+                const options = {
+                  target: element,
+                };
+
+                control.preLift(wrapper, options);
+                control.lift(wrapper, options);
+                control.end(wrapper);
 
-              control.preLift(wrapper, options);
-              control.lift(wrapper, options);
-              control.end(wrapper);
+                expect(callbacksCalled(callbacks)({
+                  onLift: index + 1,
+                  onDrop: index + 1,
+                })).toBe(true);
+              });
+            });
+
+            it('should start a drag if the target is not an interactive element', () => {
+              const nonInteractiveTagNames: TagNameMap = {
+                a: true,
```

Comment:
Just out of interest, is there any reason why anchors are not classed as interactive elements?

Reply:
Their only interaction is a click. We actually allow draggables and drag handles to be anchors - so they can be both clickable as well as draggable. We manage this interaction separately. We completely support an anchor being an anchor as well as draggable. This feels right too. The other elements are more form elements and do not make sense as a thing if you cannot interact with it directly.

Does that help?